external-crates/move/crates/move-model-2/src/summary.rs

```diff
@@ -702,8 +709,15 @@ impl Fields {
             N::VariantFields::Empty => return,
         };
         self.positional_fields = is_positional;
+        let pos_name_of = |sym| {
+            if self.positional_fields {
+                Symbol::from(format!("pos{}", sym))
```

Comment:
ditto from above

Reply:
It's not letting me comment on the other one above this one for some reason so commenting here...

I was on the fence on this, however I think it makes sense to do this here as opposed to pushing it into the model. My reasoning being that in a bytecode-model we don't _know_ that a field is positional or not (or if someone just named their field pos0...), so introducing this distinction would just introduce some drift between binary and source models that don't really need to be there.